terraform/cluster/aws-eks/variables.tf

```diff
@@ -0,0 +1,126 @@
+
+#-----------------------------------------------------------------------------------------------------------------------
+# Variables
+#-----------------------------------------------------------------------------------------------------------------------
+
+variable "context_path" {
+  type        = string
+  description = "The path to the context folder, where kubeconfig is stored"
+  default     = ""
+}
+
+variable "context_id" {
+  type        = string
+  description = "The windsor context id for this deployment"
+  default     = ""
+}
+
+variable "cluster_name" {
+  description = "The name of the EKS cluster."
+  type        = string
+  default     = ""
+}
+
+variable "kubernetes_version" {
+  description = "The kubernetes version to deploy."
+  type        = string
+  default     = "1.32"
```

Comment:
Add a `# renovate: ` tag so that we get the latest k8s from `kubernetes/kubernetes` github releases